### DISCUSSION TITLE
Shoutrrr send update IP!

### DISCUSSION THREAD

**Discussion by akha666** [qdm12/ddns-updater#616] [Q&A] [answered]
How to configure Shoutrrr to send the updated IP only once it changed? I have a notification every 10 minutes. configured with Telegram Bot, and it's very annoying.

**Comment by qdm12** [comment]
What's the notification text you get?
Also what version are you running?

**Reply by akha666** [reply]
What's the notification text you get?
DDNS Updater
my.ddns-domain.com changed to Public_IP
![ddns](https://github.com/qdm12/ddns-updater/assets/28590461/36128e97-7947-4c22-b055-08ca0feee0c7)

 Also what version are you running?
the latest one. I don't know how to get the version that I use!

**Reply by qdm12** [reply]
> I don't know how to get the version that I use!

Check the logs, it should be written at the top.

I'm suspecting it's an old version that would trigger every update? Feel free to also share your logs that would help understand!

**Comment by akha666** [comment, marked as answer]
many thank for your point. Acutally I've found the trick. I use DNSCrypt in my home OPNSense, and I have an override records which is point to my local service. 
The "RESOLVER_ADDRESS" helps me to resolve from the public DNS instead of using my local records.
The Q: How can I use multiple resolver in the env?

**Reply by qdm12** [reply]
So far it's only a single resolver address, why do you want to use multiple?

**Comment by akha666** [comment]
Just for HA Resolving and query latency

**Reply by qdm12** [reply]
> HA Resolving 

How would you see it? Trying a resolver after another in order, in case of failure? Feel free to comment on the new issue #622 

> query latency

This could be feasible, we could re-order the resolvers to try according to the first response latency, but is this really necessary though 🤔 (just trying to not-overcomplicate the code more than needed). Again let's continue on #622 😉

**Reply by akha666** [reply]
Thank you, I appreciate your work :+1:
